report/sections/Frontend/client-server-interaction.tex

```diff
@@ -14,17 +14,19 @@ \section{Client-Server Interaction}
 
 In the interest of reducing the number of database queries, the result of this request is cached on the client.
 The client will eagerly request new data to stay up to date with instructions or test code updates.
-We deemed that this short cache storage time is useful for when lecturers update test code during an exercise session.
+This short cache storage time is useful for when lecturers update test code during an exercise session.
 This way, all students have the latest test code shortly after the update while still minimizing unnecessary queries.
 
-When a user submits their exercise solution attempt, a mutation is sent to the Next.js backend.
-In this request, the necessary code and test code is included.
+When a user submits an exercise solution, a HTTP POST request is sent to the Next.js backend.
```

Comment:
I really disagree with changing mutation to HTTP POST request. We're using tRPC, so we use the terminology it uses.
Carlos wrote `what do you mean?` in his feedback, which tells me that he didn't read the preliminaries, where mutations are explained.

Reply:
maybe a compromise: a mutation is sent to the backend component in the form of an HTTP POST request.

Reply:
No, that confuses the terminology

Reply:
```suggestion
When a user submits their exercise solution attempt, a mutation is sent to the Next.js backend.
```